tests/publiccloud/check_services.pm

```diff
@@ -24,6 +24,11 @@ sub run {
     # Debug
     $instance->ssh_script_run('systemctl --no-pager list-units');
 
+    # Check if there are any failed services
+    my $failed_services = $instance->ssh_script_output('systemctl --failed');
+    die("There are some failed services!\n$failed_services")
+      unless ("0 loaded units listed" =~ $failed_services);
+
```

Comment:
Looks like a duplication of https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/6a729f1e5c91f9fa26c61edc46af9ea7ec1380f9/lib/publiccloud/instance.pm#L472